web/pano_editor.css

```
@@ -909,29 +901,35 @@
-  stroke: currentColor;
-  fill: none;
-  stroke-width: 1.5;
-  stroke-linecap: round;
-  stroke-linejoin: round;
-}
+.pano-btn-texticon svg rect:not([fill]),
+.pano-btn-texticon svg circle:not([fill]),
+.pano-btn-texticon svg line:not([fill]),
+.pano-btn-texticon svg polyline:not([fill]),
+.pano-btn-texticon svg polygon:not([fill]) {
+  stroke: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)